config/locales/ht.yml

```diff
@@ -0,0 +1,144 @@
+  last_hour: "Dènye lè "
+  last_day: "Dènye jou"
+  last_week: "Semèn pase"
+  last_month: "Mwa pase"
+  last_year: "Ane pase"
+  custom_range: 'Peryòd pèsonalize'
+  from: 'Soti'
+  to: 'Rive'
+# It's safe to modify the order of these (e.g., '%m/%d/%Y'), but do not localize the Latin letters 'd', 'm', or 'Y'. Do not change the case of the letters.
+  cdr_format: '%d/%m/%Y'
+  before: 'Anvan'
+  clear: 'Efase'
+
+# Prompts for spelling, spelling overrides, and did you mean suggestions
+  showing_results_for: "Men rezilta yo pou %{corrected_query}"
+  search_instead_for: "Chèche de preferans pou %{original_query}"
+
+# Prompts for searchers to revise or refine search
+  no_results_for_and_try: "Dezole, nou pa jwenn okenn rezilta pou %{query}. Eseye mete mwens mo oswa mo rechèch ki pi jeneral."
+  too_long: "Mo rechèch ou twò long. Eseye ankò men mete yon mo oswa yon fraz ki pi kout"
+  empty_query: "Tanpri ekri mo rechèch nan bwat ki anlè a."
+
+# Labels for inline best bets, news, and related searches modules
+  recommended: "Rekòmande"
+  default_rss_govbox_label: 'Nouvèl'
+  related_searches: "Rechèch ki an rapò yo"
+
+# Labels for paging through results
+  page: 'Paj'
+  prev_label: "Presedan"
+  next_label: "Pwochen"
+
+  searches:
+    by: "pou"
+# Prompt for searchers to skip to main content (most easily seen by tabbing through results page)
+    skip_to_main_content: "Sote pati sa a pou ale nan kontni prensipal la"
+# Labels for navigation bar "sections" (most easily seen on mobile dimension)
+    menu: "Meni"
+    browse_site: "Vizite sit la"
+    related_sites: "Gade sijè yo"
+    results_count:
+      zero: '0 rezilta yo'
+      one: '1 rezilta yo'
+      other: '%{formatted_count} rezilta yo'
+    search_results: "Rezilta rechèch yo"
+# Prompts for searchers to view more results from Bing/Google after reaching page n of Search.gov results
+    commercial_results:
+      find_what_looking_for: "Èske w reyisi jwenn sa w t ap chèche a?"
+      see_more_web_results: "%{link} pou wè plis rezilta. (Ou gendwa wè kèk nan rezilta w te wè deja.)"
+      see_more_image_results: "Rezilta ki anlè yo soti nan kont medya sosyo ak nan galri nou yo. %{link} pou wè rezilta yo ki sou wèbsayt nou an. (Ou gendwa wè kèk nan rezilta w te wè deja.)"
+      search_again: "Refè rechèch la ankò"
+# Prompts for searchers to view more results from the broader website after initially seeing site-scoped results
+    site_limits:
+      including_results_for_query_from_matching_sites: "N ap enkli rezilta yo pou %{query} ki soti nan %{matching_sites} sèlman."
+      do_you_want_to_see_results_for: "Èske w vle wè rezilta yo pou %{query_from_all_sites}?"
+      query_from_all_sites: '%{query} ki soti tout kote'
+# Label for inline news module and prompt to see more news on news vertical
+    more_news_about_query: 'Plis %{news_label} konsènan %{query}'
+    news_about_query: '%{news_label} konsènan %{query}'
+    news_search_options: "Sijè ki an rapò yo"
```

Comment:
@SamiraMSadat it turns out we do not want a localization on this line. Please remove, so it just says `news_search_options:`